src/components/SideTray/style.css

```diff
@@ -1,5 +1,5 @@
-.sideTray {
-  width: var(--rvr-side-tray-width);
```

Comment:
Why are we losing the variable here?

Reply:
Hey @mdespuits,

I suggested this. I figured that:
1. The sidetray width isn't intended for re-use among components. 
2. There's nothing magical about 400 px in our sizing system.
3. It's not part of the theme.

I could see us standardizing on some sizes for modals / popovers / etc. in the future, but for now I thought this was cleaner than having a block of sidebar-specific properties in our global variables.